go/billing/tasks.py

```diff
@@ -357,6 +357,36 @@ def generate_chunks(item_iter, items_per_chunk=1000, sep="\n"):
     return archive
 
 
+@task
+def gen_statement_then_archive(account_id, from_date, to_date, delete=True):
+    """Task to generate a monthly statement for the given account,
+    then archive the account.
+    """
+    generate_monthly_statement(account_id, from_date, to_date)
+    archive_transactions(account_id, from_date, to_date, delete)
```

Comment:
Maybe we should chain these tasks as mentioned in http://celery.readthedocs.org/en/latest/userguide/tasks.html#avoid-launching-synchronous-subtasks ?